hotshot-state-prover/src/mock_ledger.rs

```diff
@@ -347,21 +362,16 @@ impl MockLedger {
     pub fn get_state(&self) -> ParsedLightClientState {
         // The ugly conversion due to slight difference of `LightClientState` in solidity containing `threshold`
         let pi = vec![
-            u256_to_field(self.threshold),
             F::from(self.state.view_number as u64),
             F::from(self.state.block_height as u64),
             self.state.block_comm_root,
-            self.state.fee_ledger_comm,
-            self.state.stake_table_comm.0,
-            self.state.stake_table_comm.1,
-            self.state.stake_table_comm.2,
         ];
         let pi: GenericPublicInput<F> = pi.into();
         pi.into()
     }
 
     /// Returns the (bytes32 votingStakeTableComm, bytes32 frozenStakeTableComm) used in contract
-    pub fn get_stake_table_comms(&self) -> (H256, H256) {
+    pub fn get_stake_table_comms(&self) -> (H256, H256, ParsedStakeState) {
```

Comment:
is there a reason why we still return the first two commitments?
maybe renamed to `get_stake_table_state(&self) -> ParsedStakeTableState`